tests/compute_ref_sol.py

```diff
@@ -0,0 +1,25 @@
+import os
+import json
+import pysvzerod
+
+def compute_all_ref_sol():
```

Comment:
I assume this function is not being run as part of the tests and is just there to make it easy to add new reference solutions? I think this function might be dangerous in its current form. If a developer implements something that accidentally changes the results of other tests, this function will overwrite all the old (probably correct) reference solutions with the new (accidentally changed) results. Those new result will then pass all the tests because the reference solutions themselves were changed, and the developer might never realize they had changed things they were not intending to. 

However, it is very useful to have a similar function so developers can save data in the correct format for reference solutions of new tests they add (or old tests that they are _**intentionally**_ changing the results of). So can you change this to compute the reference solution for a specified test case (which can be the function argument) instead of iterating through all test cases? And print a statement at the end saying something like "the reference solution was saved here (the file path). Please verify that the results are as expected". Just to remind a developer to verify the reference solution before pushing, because previously we manually put in the numbers that the results were compared against so it was hard to not verify it.

Reply:
addressed this in the most recent commit!